packages/apps-config/src/settings/endpoints.ts

```diff
@@ -17,6 +17,27 @@ interface EnvWindow {
   }
 }
 
+function createOwn (t: TFunction): LinkOption[] {
+  try {
+    const CUSTOM_ENDPOINT_KEY = 'polkadot-app-custom-endpoints';
```

Comment:
This needs to be shared between this package and apps. I was thinking about setting and exporting it in this package, and pull if in apps from `@polkadot/apps-config/settings`. How does this sound?